src/Accordion/Accordion.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 type AccordionProps = React.HTMLAttributes<HTMLDivElement>;
 
 const Accordion = ({ ...rest }: AccordionProps): JSX.Element => {
-    return <div {...rest} />;
+    return <div data-accordion {...rest} />;
```

Comment:
I would quite like these data attributes to follow a convention, so that they're named consistently throughout, but only have a different value. Like `data-accordion-role="accordion"`, `data-accordion-role="control"` etc.

Reply:
Though maybe a synonym for 'role', like 'capacity' instead, to prevent confusion with the HTML 'role' attribute.

Reply:
How about `data-accordion-component="Accordion"` and `data-accordion-component="AccordionItemHeading"`?

Reply:
Makes sense 👍